src/utils/renderDom.ts

```diff
@@ -0,0 +1,8 @@
+import Block from './block.ts';
+
+function renderDOM(rootSelector: string, component: Block): void {
+  const root: HTMLElement = document.querySelector(rootSelector);
+  root.innerHTML = '';
+  root.append(component.getContent());
```

Comment:
`querySelector` может вернуть null
```suggestion
  const root: HTMLElement = document.querySelector(rootSelector);
  root?.innerHTML = '';
  root?.append(component.getContent());
```